doc/source/pandas_on_ray.rst

```diff
@@ -0,0 +1,71 @@
+Pandas on Ray
+======
```

Comment:
These things should be the same length as the title, so

```
Pandas on Ray
===========
```

same with the other headings throughout the page